src/pages/home/report/PureReportActionItem.tsx

```diff
@@ -1105,6 +1107,9 @@ function PureReportActionItem({
     const isWhisperOnlyVisibleByUser = isWhisper && isCurrentUserTheOnlyParticipant(whisperedTo);
     const displayNamesWithTooltips = isWhisper ? getDisplayNamesWithTooltips(whisperedToPersonalDetails, isMultipleParticipant) : [];
 
+    if (isActionableWhisper && isActionableWhisperResolved) {
```

Comment:
Can you add a comment with a short explanation here to understand why we do this?